ios/tools/sourcery/templates/Shared.stencil

```diff
@@ -0,0 +1,23 @@
+import Shared
+import KMPNativeCoroutinesCombine
+import Combine
+
+{% for type in types.protocols where type.name | hasSuffix:"UseCase" %}
+  {% for method in type.methods where method.name | hasPrefix:"invokeAsNative(" %}
+extension {{ type.name }} {
+
+  func invokeAsPublisher(
```

Comment:
Can we use `callAsFunction`?

Reply:
@suho I still prefer to use `invokeAsPublisher` due to we can support to use `Async` so we can have this function `invokeAsAysn` 🙏

Reply:
@markgravity We are converting Kotlin to Swift, and also, for `UseCase`, we should omit the function name. In Kotlin, they use `invoke` operator to omit the function name. In Swift, we have `callAsFunction`, please use it. (Check android doc: https://developer.android.com/topic/architecture/domain-layer#use-cases-kotlin)

Reply:
@suho as we discussed in the Slack, please ignore this naming for now, I will open another PR to rename all of them at once 🙏